ansible_collections/arista/avd/docs/how-to/first-project.md

```diff
@@ -520,18 +517,14 @@ Once your configuration files have been generated, you can use [`arista.avd.eos_
 
 ### Complete AVD eAPI playbook
 
-The overall playbook is given for inforamtion below and you can update it to create your own workflow
+The overall playbook is given for information below and you can update it to create your own workflow
 
 ```yaml
 ---
 - name: Build Switch configuration
   hosts: all
   tasks:
-    - name: build local folders
```

Comment:
Why is this task removed?
The playbook should include it in case the user manually deletes the output folder.